packages/rich-text-editor/src/RichTextEditor/RichTextEditor.tsx

```diff
@@ -56,25 +58,41 @@ export const RichTextEditor: React.VFC<RichTextEditorProps> = props => {
     classNameOverride,
     controls,
     rows = 3,
+    dataError = <p>Something went wrong</p>,
```

Comment:
I'm wondering if it makes sense to have a default, and if so - what would be best?